bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
       for Kubernetes", "multicluster engine for Kubernetes"]'
     operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
-  name: idp-mgmt-operator.v0.0.1
+  name: idp-mgmt-operator.v0.0.0-20220115-00-41-28
```

Comment:
Please revert.

Reply:
Current `make bundle` did this.  I added more to README to help remind us to revert